src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -262,7 +263,91 @@ protected set
                 {
                     foreach (View child in Children)
                     {
-                        child.ControlState = value;
+                        ControlState Allowed = child.PropagatableControlStates;
+                        if (Allowed.Contains(ControlState.All))
+                        {
+                            child.ControlState = value;
+                        }
+                        else
+                        {
+                            ControlState prevControlState = child.ControlState;
+                            ControlState newControlState = prevControlState;
```

Comment:
I'm wondering why `child.ControlState` does not set directly to `newControlState`.
Will `prevControlState` be used again next time?